sway/sway.5.scd

```diff
@@ -440,6 +440,17 @@ runtime.
 	is over an empty workspace. Using a mouse binding over a layer surface's
 	exclusive region is not currently possible.
 
+	Special events SWIPE\_{3,4}\_{UP,DOWN,LEFT,RIGHT} respond to three- and
```

Comment:
Nit: add `_` around `SWIPE\_{3,4}\_{UP,DOWN,LEFT,RIGHT}`

Reply:
We should probably also make it clearer that the number of fingers is optional here?

Reply:
@emersion Thanks for the review! I added the underscores to the man page. Currently, the finger count isn't optional though since it's part of the special "key" identifier. It could be if we went the route of having a `--fingers` option to `bindsym`. I'll look into that and moving the finger count to struct `sway_binding` to see what it feels like. I won't be able to get to that before the week after next though. Bear with me. :)

Reply:
> Currently, the finger count isn't optional though since it's part of the special "key" identifier

Ah, I see. I guess I just assumed 2-finger swipes were going to be available as well, but these are interpreted as scrolling instead.

> I won't be able to get to that before the week after next though. Bear with me. :)

No problem, we aren't in a hurry. I'm also guilty of dragging this for too long, I'll try to be more responsive in the future. :)